src/variables/utils/getWindowVars.test.ts

```diff
@@ -1,78 +1,78 @@
-import {getWindowPeriodVariable} from 'src/variables/utils/getWindowVars'
-import {defaultVariableAssignments} from 'src/variables/mocks'
+// import {getWindowPeriodVariable} from 'src/variables/utils/getWindowVars'
+// import {defaultVariableAssignments} from 'src/variables/mocks'
```

Comment:
TODO (not yet done in this WIP).
Updating/adding tests.
Have a full set of test cases with scoping rules etc.